src/index.js

```diff
@@ -78,7 +90,10 @@ async function startTemplateGenerator() {
     }
 
     if (results.type === 'custom') {
-      await generateFilesFromCustom(results)
+      const { templateName } = config
+      if (!templateName) throw new Error('Please add a templateName to the config if using the custom type')
```

Comment:
According to:[docs](https://github.com/CVarisco/create-component-app/blob/master/docs/CUSTOM-TEMPLATES.md) 
What happens if the user doesn't create a subfolder on the "custom" template dir ?
Do you understand what I mean ?

For example:

customTemplateDir
   |---index.js

```json
{
  "type":"custom",
  "template": ?????,
  "templatesDirPath": "./customTemplateDir"
}
```

Thanks a lot for you support 👍

Reply:
if the template does not exist, getTemplate will throw an error if that's what you mean

Reply:
But I didn't test what happens when the templatesDirPath itself doesn't exist

Reply:
Thanks man !
I'll merge now 🚜

Reply:
oh... so you were planning to have either one template in templatesDirPath, or several?
how would you distinguish? either way, type custom wasn't really working was it? (at least I didn't get it to work without throwing an error)

Reply:
and thanks for merging ;)

Reply:
Sure! You are right!
The first implementation was different and maybe all the changes brake something.

However, now it is better :)
Thank you 💻